convert2rhel/subscription.py

```diff
@@ -223,13 +223,17 @@ def remove_original_subscription_manager():
 
 
 def install_rhel_subscription_manager():
-    loggerinst.info("Installing subscription-manager RPMs.")
+    loggerinst.info("Checking for subscription-manager RPMs.")
     rpms_to_install = [os.path.join(SUBMGR_RPMS_DIR, filename) for filename in os.listdir(SUBMGR_RPMS_DIR)]
 
     if not rpms_to_install:
-        loggerinst.warn("No RPMs found in %s." % SUBMGR_RPMS_DIR)
+        loggerinst.warning("No RPMs found in %s." % SUBMGR_RPMS_DIR)
         return
 
+    pkg_names = get_installed_submgr_pkg_names(rpms_to_install)
+    rpms_to_untrack = check_for_pkg_presence(pkg_names)
+
```

Comment:
Can be moved below the submgr install

Reply:
Actually, these calls need to be before the submr install, we need to track any packages that are present on the system before the installation of submr, if, for example, `python3-syspurpose` or `python3-cloud-what` were already present on the system (and may be used by other packages other than submr, we need to untrack them and let those packages be present on the system after the conversion)

Reply:
You're right. I'd add a comment instead then so we don't accidentally move it in the future to be closer

Reply:
Sure! Good idea. Will add this command right now